### PR TITLE
Submission feedback

### PR DESCRIPTION
**Pull request by Valdorff**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,72 +77,686 @@ There's no significant difference.
 |:-------------------------------------------:|:--------------------------------------------------:|
 
 The plot above shows a survival function for bids on RP vanilla blocks vs non-RP vanilla blocks.
-Ideally we'd expect the same curves to be followed. They mostly are.
+Ideally we'd expect the same curves to be followed. They mostly are. As previously noted, nonRP
```
Comment (xrchz) [inline README.md]: should nonRP have a hyphen?

File: README.md [hunk 2]
```diff
@@ -9,8 +9,6 @@ We analyze 40.7 weeks of data, starting right after the MEV grace period ended a
 (2022-11-24 05:35:39Z UTC); see
 https://discord.com/channels/405159462932971535/405163979141545995/1044108182513012796.
 
-
-
 ### Global vs RP sanity check
```
Comment (xrchz) [inline README.md]: Can we avoid "sanity check" here? (E.g. https://source.android.com/docs/setup/contribute/respectful-code for why and suggested alternatives.)

File: README.md [hunk 3]
```diff
@@ -48,10 +46,12 @@ such evidence.
     likely due to chance. It's also not "worth" stealing as a penalty should be applied and will
     make this size theft a net loss.
 - Second row: if anything, it looks like vanilla blocks are underrepresented in slots with high max
-  bids. One theory to explain this would be that folks likely to use vanilla blocks don't use many
-  relays, which means they get smaller max-bids on average. Another (imo less likely) theory is that
-  we don't see the lines diverge until we're starting to get sparser, so it may simply be chance.
-- Third row: these match as well as plausible.
+  bids. theory is that since we're defining anything beyond RP-approved relays as "vanilla", then
```
Comment (xrchz) [inline README.md]: Missing "The" at the start of the sentence (that currently starts "theory")?

**Comment by xrchz** [comment]
Happy to merge if you fix all the comments.

**Review by xrchz — APPROVED** [review]

**Comment by Valdorff** [comment]
I believe I addressed all comments

**Pull request merged**